src/emulator/gxm/include/gxm/state.h

```diff
@@ -16,4 +16,5 @@ namespace emu {
 
 struct GxmState {
     emu::SceGxmInitializeParams params;
+    bool isInScene = false;
```

Comment:
Can games draw to more than one context at the same time? Maybe this variable should be in the context after all.

Reply:
No, sceGxmBeginScene can't be called if a context is already in scene. It's not possible to have multiple scenes active at the same moment.